lockdown/tests/tests.py

```diff
@@ -57,6 +57,28 @@ def test_url_exceptions(self):
         response = self.client.get(self.locked_url)
         self.assertEqual(response.content, self.locked_contents)
 
+    @patch('lockdown.tests.tests.middleware.settings.REMOTE_ADDR_EXCEPTIONS',
+           ('192.168.0.1',))
+    def test_remote_addr_exception_lock(self):
+        """Test that a page isn't locked when client IP is in exception list
```

Comment:
This docstring is the same as for the following test and doesn't seem to be correct.

Reply:
Typo - will be changed.